test-lib.sh

```diff
@@ -139,7 +139,7 @@ function ct_assert_container_creation_fails() {
 function ct_create_container() {
   local cid_file="$CID_FILE_DIR/$1" ; shift
   # create container with a cidfile in a directory for cleanup
-  docker run --cidfile="$cid_file" -d ${CONTAINER_ARGS:-} $IMAGE_NAME "$@"
+  docker run --cidfile="$cid_file" -d "${CONTAINER_ARGS:-}" $IMAGE_NAME "$@"
```

Comment:
This doesn't feel right.  Spaces in CONTAINER_ARGS are to separate options.

This is a "known" weak point though, we should have there properly shell-escaped options so the "docker run" can be executed through eval.  That way you can have multiple args and some of them can container spaces.